components/PlatformSupport.tsx

```diff
@@ -0,0 +1,47 @@
+import { hasWindowsSupport, detectPlatformInfo } from '../data/platformMapping'
+
+interface PlatformSupportProps {
+  platforms: string[]
+}
+
+export const PlatformSupport = ({ platforms }: PlatformSupportProps) => {
+  if (!platforms || platforms.length === 0) {
+    return (
+      <div className="text-gray-500 text-sm">
+        Platform support information not available
+      </div>
+    )
+  }
+
+  const hasWindows = hasWindowsSupport(platforms)
+
+  return (
+    <div className="space-y-2">
+      <h2 className="text-2xl font-bold mt-4 mb-2">Platforms</h2>
```

Comment:
"Platforms" is ambiguous in Bazel terminology - I think we should say "tested on" (it might support more than that)